parent/pom.xml

```diff
@@ -2103,6 +2133,71 @@
                </plugin>
 
                <plugin>
+                  <groupId>org.codehaus.mojo</groupId>
+                  <artifactId>build-helper-maven-plugin</artifactId>
+                  <version>1.7</version>
```

Comment:
This version should not be needed here... (already defined in `//build/pluginManagement`)